currencyii/src/main/java/nl/tudelft/trustchain/currencyii/ui/bitcoin/BitcoinFragment.kt

```diff
@@ -118,23 +155,62 @@ class BitcoinFragment : BaseFragment(R.layout.fragment_bitcoin),
             return
         }
 
-        var walletManager = WalletManagerAndroid.getInstance()
+        val walletManager = WalletManagerAndroid.getInstance()
 
-        walletBalance.text =
-            "${walletManager.kit.wallet().balance.toFriendlyString()}"
-        if (walletManager.params.id === NetworkParameters.ID_MAINNET) {
-            chosenNetwork.text = "Production Network"
-        } else {
-            chosenNetwork.text = "Test Network"
+        walletBalance.text = walletManager.kit.wallet().balance.toFriendlyString()
+        walletEstimatedBalance.text = walletManager.kit.wallet().getBalance(Wallet.BalanceType.ESTIMATED).toFriendlyString()
+        when {
```

Comment:
Can you make this a switch

Reply:
What do you mean? This when is a switch in Kotlin right?

Reply:
sorry what I meant was:
```
when (walletManager.params.id) {
  NetworkParameters.ID_MAINNET -> ...
  NetworkParameters.ID_REGTEST -> ...
  NetworkParameters.ID_TESTNET -> ...
}
```